### PR TITLE
Beltran's frontend work 4/22

### PR DESCRIPTION
**Pull request by hgzimmerman**

### CODE REVIEW

File: frontend/src/app/home/containers/Home.js [hunk 1]
```diff
@@ -51,12 +51,27 @@ export default class Home extends React.Component {
 
   }
 
+  sellStock(stock, quantity, price){
```
Comment (hgzimmerman) [inline frontend/src/app/home/containers/Home.js]: What is this?

File: frontend/src/app/stocks/containers/Stocks.js [hunk 2]
```diff
@@ -29,22 +29,59 @@ export default class Stocks extends React.Component {
...
+      alert('Please enter a valid stock')
+    }
+
+
+  }
+
+  buyStock(stock, quantity, price){
+    const url = '/api/market/stock/transact';
+    const headers = {
+      'Authorization: ': 'bearer token'
```
Comment (hgzimmerman) [inline frontend/src/app/stocks/containers/Stocks.js]: `token` in this line should be replaced with the contents of window.localstorage.getItem('jwt')

File: frontend/src/app/stocks/containers/Stocks.js [hunk 3]
```diff
@@ -29,22 +29,59 @@ export default class Stocks extends React.Component {
...
+    }
+    const body = JSON.stringify({
```
Comment (hgzimmerman) [inline frontend/src/app/stocks/containers/Stocks.js]: Wrong body type for this request. instead you want to reference this structure:
```
#[derive(Serialize, Deserialize, Clone, Debug)]
pub struct StockTransactionRequest {
    /// The stock symbol.
    pub symbol: String,
    /// The sign bit indicates if it is a sale or a purchase;
    pub quantity: i32,
}
```

File: frontend/src/app/home/containers/Home.js (hunk 1 above)
Comment (beltrancaliz) [reply frontend/src/app/home/containers/Home.js]: Function for selling owned stocks that are displayed on the home component

File: frontend/src/app/stocks/containers/Stocks.js (hunk 2 above)
Comment (beltrancaliz) [reply frontend/src/app/stocks/containers/Stocks.js]: Added to next commit

File: frontend/src/app/stocks/containers/Stocks.js (hunk 3 above)
Comment (beltrancaliz) [reply frontend/src/app/stocks/containers/Stocks.js]: Is it x-www-form-urlencoded body type?

Comment (hgzimmerman) [reply frontend/src/app/stocks/containers/Stocks.js]: no, body type should be json in all cases (unless I misunderstand). The only urlencoding is done on one particular get query params.

**Pull request merged**